### PR TITLE
Fhmap 64 app read by

### PR DESCRIPTION
**Pull request by david-martin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,5 +134,5 @@ This is useful when you need to do a list domains before deciding which one to u
     
     export  OPENSHIFT_USERNAME="unamehere"
     export OPENSHIFT_PASSWORD="pwdhere"
-    ./node_modules/.bin/turbo test/accept.js
```
Comment (cianclarke) [inline README.md]: Good idea

File: test/fixtures.js [hunk 2]
```diff
@@ -17,6 +17,8 @@ module.exports = nock('https://openshift.redhat.com')
 .reply(200, datareply, headers)
 .get('/broker/rest/cartridges', '*')
 .reply(200, datareply, headers)
+.get('/broker/rest/application/1a')
+.reply(200, datareply, headers)
```
Comment (cianclarke) [inline test/fixtures.js]: @david-martin what do you make of these nock-like fixtures? 
I find them a bit of a pain to debug, but a lot tidier than excessive proxyquiring of the request module

**Comment by cianclarke** [comment]
Reviewed, thanks Dave

File: test/fixtures.js (hunk 2 above)
Comment (david-martin) [inline test/fixtures.js]: seem ok, but hard to maintain and debug.
alternative is to mock expected responses as required in test functions

(forgive mockito/jasmine combined syntax)
e.g. 
```
when(request).get(<dont care>).then(function() {
  return cb(null, mockRes);
});
<test code here>
```

not sure what would provide something like this

**Pull request merged**